code/src/App.js

```diff
@@ -1,9 +1,30 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { Thoughts } from './components/Thoughts';
 
 export const App = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [thoughtsArray, setThoughtsArray] = useState([]);
+
+  const API_URL = 'https://happy-thoughts-ux7hkzgmwa-uc.a.run.app/thoughts'
```

Comment:
Maybe it's an obvious one, but making the API link a const (like you did), makes the code nicer and more professional :)